core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -35,101 +35,103 @@ import scalaz._, Scalaz._
   * @groupdesc dist Natural transformations required for generalized folds and unfolds.
   */
 package object matryoshka {
+  type Fix[F[_]] = matryoshka.data.Fix[F]
+  val Fix        = matryoshka.data.Fix
```

Comment:
I must have done those Fix changes in response to something which broke from the other commits, because all I wanted out of this was the plugin updates. Hmmm, don't remember.